### DISCUSSION TITLE
Golang 1.21 download issue

### DISCUSSION THREAD

**Discussion by igormiku** [renovatebot/renovate#23822] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

Hello

Seems Renovate is unable to download automatically latest golang as tool via 
https://dl.google.com/go/go1.21.0.linux-amd64.tar.gz

It attempts https://dl.google.com/go/go1.21.linux-amd64.tar.gz, which is not redirected by google to 1.21.0 by some reason.

Perhaps, issue also lays with Google itself with redirect.

Note that https://dl.google.com/go/go1.20.linux-amd64.tar.gz works

### Relevant debug logs

```
Command failed: install-tool golang 1.21.0
curl: (22) The requested URL returned error: 404 
Download failed: https://dl.google.com/go/go1.21.linux-amd64.tar.gz

```

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
Pease retry with the latest version

**Reply by igormiku** [reply]
We run 36.41, didn't find any commit in next versions which solves related problem. Can you please confirm that?

**Reply by igormiku** [reply]
```
Command failed: install-tool golang 1.21.0
curl: (22) The requested URL returned error: 404 
Download failed: https://dl.google.com/go/go1.21.linux-amd64.tar.gz

```

**Reply by igormiku** [reply]
https://github.com/containerbase/base/commit/803978dca26c3e40134f278cc23f3fddc3bd7353

**Reply by viceice** [reply]
are you using the official renovate image?

**Reply by igormiku** [reply]
Yes

**Reply by viceice** [reply]
you're definitely not using latest renovate version, as it no longer uses curl to download golang.

please ensure to pull the latest renovate version
